### PR TITLE
Bug 1505869 - Use the distinclude map to improve analysis for files c…

### PR DESCRIPTION
**Pull request by staktrace**
…opied into dist/include.

On Windows, files are copied, rather than symlinked, into dist/include.
This can result in the analysis data ending up in the
`__GENERATED__/dist/include` folder when it should really be at the path
corresponding to the original source location of the header. This patch
uses the distinclude mapfile to correct that.

### CODE REVIEW

File: mozilla-central/setup [hunk 1]
```diff
@@ -179,7 +192,55 @@ if [ -n "$INDEXED_GIT_REV" ]; then
             fi
         done
         set -x
+        popd
+
+        date
+
+        # On Windows, analysis for headers ends up in __GENERATED__/dist/include/...
```
Comment (amccreight) [inline mozilla-central/setup]: These comments only talk about Windows, but it sounds like this is done for all platforms? Are there files you know that this helps, or is it only for completeness?

File: mozilla-central/setup [hunk 2]
```diff
@@ -179,7 +192,55 @@ if [ -n "$INDEXED_GIT_REV" ]; then
...
+                fi
+            fi
+        done
+        set -x
+        popd
+
+        date
+
         # Also drop any directories that got emptied as a result
+        pushd analysis-$PLATFORM/__GENERATED__
```
Comment (amccreight) [inline mozilla-central/setup]: Why is this change needed?

File: mozilla-central/setup (hunk 1 above)
Comment (staktrace) [reply mozilla-central/setup]: It only helps on windows, but I did it for all platforms for completeness.
